package.json

```diff
@@ -32,28 +32,28 @@
     },
     "dependencies": {
         "@eslint-community/eslint-utils": "^4.4.0",
-        "globals": "^14.0.0",
-        "rambda": "^7.4.0"
+        "globals": "^15.9.0",
+        "rambda": "^9.2.1"
```

Comment:
Should we use [exact](https://docs.npmjs.com/cli/v10/commands/npm-install#save-exact) dependency versions here like in others (but not all) dependencies?

Reply:
Since this package is not published very frequently I would like to keep version ranges in regular dependencies. The `devDependencies` should be already exact versions.

Reply:
👀 at `@dprint/typescript` and `dprint`

Reply:
Fixed. PTAL